rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -811,6 +817,18 @@ private void processReleaseTransactions(Transaction rskTx) {
         }
     }
 
+    private void updateFederationCreationBlockHeights() {
+        if (!activations.isActive(RSKIP186)) {
+            return;
+        }
+
+        Long nextFederationCreationBlockHeight = provider.getNextFederationCreationBlockHeight();
+        if (nextFederationCreationBlockHeight != null) {
```

Comment:
I'm not sure about using null as an "not present" value. Usually, an smart contract using storage finds the unused cell already initialized in zero.

Reply:
`null` is only being used at the data level - to distinguish if it was set. Precompiled contract returns `zero`, if not set.